pkg/ekcoops/operator.go

```diff
@@ -127,19 +127,32 @@ func (o *Operator) reconcileRook(nodes []corev1.Node, doFullReconcile bool) erro
 		readyCount, err := o.ensureAllUsedForStorage(nodes)
 		if err != nil {
 			multiErr = multierror.Append(multiErr, errors.Wrapf(err, "ensure all ready nodes used for storage"))
-		} else {
+		}
+
+		if readyCount > 0 {
```

Comment:
Why did you change this? It seems to me like you're relying on the return value of a function that returned error now?